doc/plutus-core-spec/introduction.tex

```diff
@@ -18,3 +18,51 @@ \section{Introduction}
 untyped Plutus Core is used as a compilation target, and we will eventually
 provide a specification of the type system and semantics of Typed Plutus Core
 here as well, together with its relationship to Untyped Plutus Core.
+
+\subsection{Terminology and version numbers}
+\label{sec:version-numbers}
+Once a Plutus Core script has been deployed to the blockchain, it may be
+re-evaluated at any point in the future when the history of the chain is
+verified.  Plutus Core may evolve as time progresses, and because of this we
+have to be quite careful with versioning.  A detailed discussion of this issue
+can be found in~\cite{CIP-35}, but we reproduce two definitions here for
+convenience.
+\begin{itemize}
+\item The language itself may evolve: for example some fundamental new construct
+  may be added.  A \textit{Plutus Core language version} describes a version of
+  the basic language with a particular set of features. A language version
+  consists of three non-negative integers separated by decimal points, for
```

Comment:
Suggestion: "non-negative integers" => "natural numbers"

Reply:
> Suggestion: "non-negative integers" => "natural numbers"

Some people think that 0 isn't a natural number, although they're probably not computer scientists.  We do say that ℕ includes zero earlier on though, so maybe that's good enough.